model-optimizer/extensions/back/SpecialNodesFinalization_test.py

```diff
@@ -150,12 +150,51 @@ def test_only_consumer(self):
         self.assertNotIn('const_data', graph.node)
         self.assertNotIn('result_node', graph.node)
 
+
+    def test_only_consumer_keep_result(self):
+        """Result node is only consumer of Const data node"""
+        nodes = [
+            ('const_node', {'type': 'Const', 'kind': 'op'}),
+            ('const_data', {'kind': 'data', 'value': np.array(5)}),
+            ('result_node', {'type': 'Result', 'kind': 'op', 'keep_output_port': True}),
```

Comment:
This example graph is a bit confusing. It describes not possible situation when we have Const->Result graph and the Result node is marked with `keep_output_port` attribute. I recommend to update the test and have a real case like:
```
 Parameter->ShapeOf->Split(n=2)->Result(keep_output_port = True)
                         \->Abs()->Result()
```